charts/kargo/templates/api/cluster-role.yaml

```diff
@@ -19,6 +19,28 @@ rules:
       - patch
       - update
       - delete
+  - apiGroups:
+      - ""
+    resources:
+      - serviceaccounts
+    verbs:
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - users
+      - groups
+      - serviceaccounts
+    verbs:
+      - impersonate
+  - apiGroups:
+      - authorization.k8s.io
+    resources:
+      - selfsubjectaccessreviews
+    verbs:
+      - create
```

Comment:
This permission, afaik, doesn't ever have to be explicitly granted. All users are able to perform selfsubjectaccess reviews.